src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,69 @@
+package duke;
+
+import duke.exception.InvalidCommandException;
+import duke.exception.MissingDescriptionOrDateException;
+import duke.task.Task;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+
+
+public class Duke {
+    private static TaskManager taskManager = new TaskManager();
+
+    public static void main(String[] args) {
+        printWelcomeMessage();
+
+        handleUserInput();
+
+        printGoodbyeMessage();
+    }
+
+    public static void printWelcomeMessage() {
+        System.out.println("____________________________________________________________");
+        System.out.println("Botty McBotface, at your service.");
+        System.out.println("Your wish is my command.");
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void handleUserInput() {
+        String userInput;
+        String[] processedInput;
+        Scanner in = new Scanner(System.in);
+
+        ArrayList<Task> taskList = new ArrayList<Task>();
```

Comment:
You can consider using plural for the name of this variable since it's an ArrayList of many things